src/Administration/Resources/app/administration/src/app/component/form/sw-text-editor/index.js

```diff
@@ -349,7 +349,7 @@ Component.register('sw-text-editor', {
         value: {
             handler() {
                 if (this.$refs.textEditor && this.value !== this.$refs.textEditor.innerHTML) {
-                    this.$refs.textEditor.innerHTML = '';
+                    this.$refs.textEditor.innerHTML = this.value;
```

Comment:
does this make sense @Haberkamp ?

Reply:
FYI @amenk Jannis is here to help with _that_ _specific_ questions cause he's more knowledgable than me regarding the text editor.

Reply:
@jleifeld Thanks for looking into this. The problem I had, was that the text editor lost content when the value is updated, i.e. when the revert-content button of this new feature was used.
I have absolutely no clue if this causes side effects, but this change fixed it 🤞
Also the part `innerHTML = ''` looks at first glance a bit like some workaround to me. But you are the expert on this. 😄

Reply:
If we change this we need to do much manual testing to check if everything still works like before. Because this change changes the main logic from the value handler which does resetting the innerHTML beforehand to now the new value. @Haberkamp for me it is okay when some QA do much manual testing with this change and try to check every edge case